src/Logger.php

```diff
@@ -68,4 +70,12 @@ public static function getDefaultLogHandler(): StreamHandler
         $logHandler->setFormatter(new LineFormatter("%message%\n"));
         return $logHandler;
     }
+
+    public static function getDefaultCriticalHandler(): StreamHandler
+    {
+        $handler = new StreamHandler('php://stderr');
+        $handler->setBubble(false);
+        $handler->setLevel(MonologLogger::CRITICAL);
```

Comment:
Možná bych i tady explicitně nastavil ten formatter ať je na první pohled jasné jaký je v tom rozdíl. Taky se oproti defaultu může odebrat `channel` který je pořád stejný (`php-component`) a nepracujeme s ním.
```php
 $handler->setFormatter(new LineFormatter("[%datetime%] %level_name%: %message% %context% %extra%\n"));
```
Takže místo:
```
[2018-05-15 10:40:50] php-component.CRITICAL: ErrorException:file_get_contents(/data/config.json): ...
```
To vyplivne:
```
[2018-05-15 10:44:14] CRITICAL: ErrorException:file_get_contents(/data/config.json):...
```
Kontext o jakou šlo appku atd je k tomu pak doplněný před zalogováním do  PT docker runnerem.

Reply:
Dobrý nápad. 

`[2018-05-15 13:20:06] CRITICAL: InvalidArgumentException:msg {"errFile":"W:\\keboola\\php-component\\example\\run.php","errLine":11,"errCode":0,"errTrace":"#0 {main}","errPrevious":""} []
`